client/components/departments_slide.js

```diff
@@ -1,8 +1,19 @@
 import React, {Component} from 'react'
 import {Link} from 'react-router-dom'
 import {Carousel} from 'react-bootstrap'
+import {connect} from 'react-redux'
+
+const Departments_slide = ({departments}) => {
```

Comment:
Can we rename this component to be `DepartmentsSlide `?

Let's also rename the file to be `departmentsSlide`
and update any usages throughout the app.

To make sure we update this throughout the site, we can use [grep](https://en.wikipedia.org/wiki/Grep)
Basically, `git grep "expression"` will return all instances of "expression" throughout the codebase.
See below how I use it
```
andres@Andress-MBP boilermaker % git grep "departments_slide"
client/routes.js:import Departments_slide from './components/departments_slide'
andres@Andress-MBP boilermaker %
andres@Andress-MBP boilermaker %
andres@Andress-MBP boilermaker % git grep "Departments_slide"
client/components/departments_slide.js:const Departments_slide = () => {
client/components/departments_slide.js:export default Departments_slide
client/routes.js:import Departments_slide from './components/departments_slide'
client/routes.js:        <Route exact path="/" component={Departments_slide} />
public/style.css:/* below are for component Departments_slide */
andres@Andress-MBP boilermaker %
```

Reply:
yes, changed.